### PR TITLE
Subtask/ucd 1323 fwa check your answers

### PR DESCRIPTION
**Pull request by a6software**
## Ticket Number

UCD-1323

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [x] My commit history in this PR is linear
- [x] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/forms-web-app/src/controllers/appellant-submission/check-answers.js [hunk 1]
```diff
@@ -15,5 +16,8 @@ exports.getCheckAnswers = async (req, res) => {
   res.render(VIEW.APPELLANT_SUBMISSION.CHECK_ANSWERS, {
     appealLPD,
     appeal,
+    otherDocuments: multifileUploadFileNamesOnly(
```
Comment (a6software) [inline packages/forms-web-app/src/controllers/appellant-submission/check-answers.js:19]: I might move this to a filter, like everything else...

File: packages/forms-web-app/src/lib/filter-by-key.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+module.exports = (dict, key) =>
```
Comment (a6software) [inline packages/forms-web-app/src/lib/filter-by-key.js:1]: comment needed

File: packages/forms-web-app/src/lib/add-key-value-pair.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+module.exports = (dict, key, value) => {
```
Comment (a6software) [inline packages/forms-web-app/src/lib/add-key-value-pair.js:1]: comment needed

File: packages/forms-web-app/src/lib/multifile-upload-file-names-only.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+module.exports = (multifileUploadsArray) =>
```
Comment (a6software) [inline packages/forms-web-app/src/lib/multifile-upload-file-names-only.js:1]: comment needed

File: packages/forms-web-app/src/views/macros/multifile-upload-to-list.njk [hunk 5]
```diff
@@ -0,0 +1,23 @@
+{% from "./summary-field.njk" import summaryField %}
```
Comment (a6software) [inline packages/forms-web-app/src/views/macros/multifile-upload-to-list.njk:1]: comments

File: packages/forms-web-app/tests/unit/controllers/appellant-submission/check-answers.test.js [hunk 6]
```diff
@@ -26,6 +26,7 @@ describe('controllers/appellant-submission/check-answers', () => {
       expect(res.render).toHaveBeenCalledWith(VIEW.APPELLANT_SUBMISSION.CHECK_ANSWERS, {
         appealLPD: '',
         appeal,
+        otherDocuments: [],
```
Comment (a6software) [inline packages/forms-web-app/tests/unit/controllers/appellant-submission/check-answers.test.js:29]: could be removed with a filter

**Review by frank-dodion — APPROVED** [review]
Approved to get implementation into feature branch to assist with glue code dev

**Pull request merged**